conjure-java-jaxrs-client/src/main/java/com/palantir/conjure/java/client/jaxrs/AbstractFeignJaxRsClientBuilder.java

```diff
@@ -180,28 +180,35 @@ public int hashCode() {
     }
 
     private static Contract createContract() {
-        return new EndpointNameHeaderEnrichmentContract(
-                new PathTemplateHeaderEnrichmentContract(new SlashEncodingContract(new Java8OptionalAwareContract(
```

Comment:
AFAICT, the `hr-path-template` header is, and only ever has been, used by the `OkhttpTraceInterceptor`. This was introduced in https://github.com/palantir/conjure-java-runtime/pull/577.

We no longer use OkHttp clients here and this header was an implementation detail, so I think we're good to remove `PathTemplateHeaderEnrichmentContract` and the related logic in `DialogueFeignClient`.